configure.ac

```diff
@@ -2388,7 +2397,7 @@ fi
 
 if test "x$with_libdpdk" = "xyes"; then
   SAVE_LDFLAGS="$LDFLAGS"
-  LDFLAGS="$LIBDPDK_LDFLAGS $LDFLAGS"
+  LDFLAGS="$LIBDPDK_LDFLAGS $LIBDPDK_LIBS $LDFLAGS"
```

Comment:
Indeed it does check for libdpdk and call rte_eal_init with the action to finally enable dpdk.
Usage of the fixed LIBs "later on" is in the right place in the Makefile replacing the hardcoded -ldpdk (if no pkg-config is found -ldpdk is set) - which in turn is a linker script expanding to the long set.
It is here only set on the LDFLAGS in between being saved and restored in case there are extra args in the provided LIBDPDK_LIBS like the include-dir to make things work.

For the current case we could drop that change and just let the linker script do it's work.
But I'd keep the replacement of the hardcoded -ldpdk in the Makefile to take -ldpdk as default but get overwritten by pkg-config if provided (or a user setting).